### ISSUE TITLE
Shift Enter breaks tmux navigator

### ISSUE THREAD

**Issue by anichols-ht** [christoomey/vim-tmux-navigator#344]
First of all . . . _love_ tmux navigator (thank you!). So much so that when something goes wrong, I have a hard time adjusting back to <ctrl-w>hjkl mappings.

I thought at first this was a problem with fern cause I started noticing it when using fern, but after a bit of debugging, it turns out that <S-CR> by itself (regardless of other plugins and whether that key combo has a mapping or not) causes tmux navigator to permanently no longer work in the vim session I'm in. I have to quit vim and relaunch to get them back (I could probably re-source it, but I haven't tried that). Seems weird, but I can reproduce it consistently.

**Comment by christoomey** [comment]
Hey @anichols-ht - thanks for the kind words!

Can you add any specifics about the failure mode, and did happen to try anything describe in the [Troubleshooting section](https://github.com/christoomey/vim-tmux-navigator#troubleshooting)?

Based on what you're describing (it works in a fresh Vim, but then stops working at some point), I'm guessing something is overriding the mappings or changing the process name.

- Can you run `:verbose nmap <C-j>` when it stops working, and compare to the output in a fresh vim instance?
- Can you confirm the output of `:TmuxNavigatorProcessList` when it stops working, and again compared to in a fresh vim session?

**Comment by anichols-ht** [comment]
Yeah, I should've included that in the initial report. `verbose nmap <c-h>` (I chose h when I tried it) still reports tmux navigator; it just doesn't work. Just checked `TmuxNavigatorProcessList` and it shows:

```
Ss  -bash
S+  vim
S+  /bin/bash
S+  tmux
```

This definitely still could be environmental or user error, but I'm not sure what else to check.

**Comment by christoomey** [comment]
Process list and mappings look right, so doesn't seem to be those.

Next things to check:

- Is the mapping getting triggered? (Run `nmap <C-j> :echo 'hello'<cr>` and then hit Ctrl-j and confirm you see `hello` printed out)
- Is the vim function not working as expected? (You can manually execute them, e.g. `TmuxNavigateRight` to run what _should_ happen when you hit Ctrl-l)

**Comment by anichols-ht** [comment]
The mapping does not get triggered and the vim functions do work, so I think that must indicate something else is the problem. Shift-Enter just seems to stop Ctrl-hjkl from working. But it doesn't prevent _other_ Ctrl bindings from working. I can still do Ctrl-w hjkl to move around (this is the weirdest part I think . . . it's so selective). I use terminal vim, so maybe it just doesn't like Shift-Enter for some reason.

**Comment by anichols-ht** [comment]
Okay. Turning off `xterm-keys` did not fix it, but removing `Send ^[ [13;2u                 Shift Return` from iterm2 key mappings did (though of course Shift-Enter no longer works for mappings).

**Comment by christoomey** [comment]
Sounds like this is coming from your xterm config rather than the plugin? Don't want to close preemptively, but assuming that's accurate, I'm thinking we can close this issue.
